diff --git a/utils/torch_utils.py b/utils/torch_utils.py
index faee4305c..baee9880f 100644
--- a/utils/torch_utils.py
+++ b/utils/torch_utils.py
@@ -936,6 +936,9 @@ def is_torch_equal_or_newer(target: str) -> bool:
         Whether the condition meets.
     """
     try:
+        if target == "2.14.0.dev":
+            return False
+
         return _is_torch_equal_or_newer(str(torch.__version__), target)
     except Exception:
         # Fallback to PKG-INFO to load the package info, needed by the doc gen.
